src/Contract/Operation/Duplicateable.php

```diff
@@ -9,6 +9,7 @@
 
 namespace loophp\collection\Contract\Operation;
 
+use Closure;
```

Comment:
found a couple of these where the import was missing but the `Closure` was mentioned in the doc blocks. Because of the missing import PHPStan couldn't resolve the type

Reply:
Indeed! Nice ones!